pywren/executor.py

```diff
@@ -185,10 +187,11 @@ def map(self, func, iterdata, extra_env=None, extra_meta=None,
             # it exceeded max data size
             pass
 
-        for module in exclude_modules:
-            for mod_path in list(mod_paths):
-                if module in mod_path and mod_path in mod_paths:
-                    mod_paths.remove(mod_path)
+        if exclude_modules:
```

Comment:
You're right. I'd assumed that a user would never pass in an empty list to `exclude_modules`, but that's not the case.